ui/DesignSystem/Primitive/Icon/TxState.svelte

```diff
@@ -93,18 +93,26 @@
       fill={color} />
   {:else if state === 'negative'}
     <path
-      d="M16.505 21C16.1072 21 15.7257 20.842 15.4444 20.5607C15.163 20.2794
-      15.005 19.8978 15.005 19.5C15.005 19.1022 15.163 18.7207 15.4444
-      18.4394C15.7257 18.158 16.1072 18 16.505 18C16.9028 18 17.2844 18.158
-      17.5657 18.4394C17.847 18.7207 18.005 19.1022 18.005 19.5C18.005 19.8978
-      17.847 20.2794 17.5657 20.5607C17.2844 20.842 16.9028 21 16.505 21ZM17.505
-      15.1C17.375 16.3 15.625 16.3 15.505 15.1L15.005 10.1C14.991 9.96054
-      15.0065 9.81967 15.0504 9.68656C15.0943 9.55344 15.1658 9.43105 15.2601
-      9.32733C15.3544 9.2236 15.4694 9.14086 15.5977 9.08447C15.7261 9.02809
-      15.8648 8.99931 16.005 9.00001H17.005C17.1452 8.99931 17.2839 9.02809
-      17.4123 9.08447C17.5406 9.14086 17.6557 9.2236 17.75 9.32733C17.8443
-      9.43105 17.9157 9.55344 17.9596 9.68656C18.0036 9.81967 18.019 9.96054
-      18.005 10.1L17.505 15.1Z"
+      d="M16 22C15.6022 22 15.2206 21.842 14.9393 21.5607C14.658 21.2794 14.5
+      20.8978 14.5 20.5C14.5 20.1022 14.658 19.7207 14.9393 19.4394C15.2206
+      19.158 15.6022 19 16 19C16.3978 19 16.7793 19.158 17.0607 19.4394C17.342
+      19.7207 17.5 20.1022 17.5 20.5C17.5 20.8978 17.342 21.2794 17.0607
+      21.5607C16.7793 21.842 16.3978 22 16 22ZM17 16.1C16.87 17.3 15.12 17.3 15
+      16.1L14.5 11.1C14.486 10.9605 14.5014 10.8197 14.5454 10.6866C14.5893
+      10.5534 14.6607 10.4311 14.755 10.3273C14.8493 10.2236 14.9644 10.1409
+      15.0927 10.0845C15.2211 10.0281 15.3598 9.99931 15.5 10H16.5C16.6402
+      9.99931 16.7789 10.0281 16.9073 10.0845C17.0356 10.1409 17.1507 10.2236
+      17.2449 10.3273C17.3392 10.4311 17.4107 10.5534 17.4546 10.6866C17.4986
+      10.8197 17.514 10.9605 17.5 11.1L17 16.1Z"
+      fill={color} />
+    <path
+      opacity="0.3"
+      fill-rule="evenodd"
+      clip-rule="evenodd"
+      d="M16 4C9.37258 4 4 9.37258 4 16C4 22.6274 9.37258 28 16 28C22.6274 28 28
+      22.6274 28 16C28 9.37258 22.6274 4 16 4ZM0 16C0 7.16344 7.16344 0 16
+      0C24.8366 0 32 7.16344 32 16C32 24.8366 24.8366 32 16 32C7.16344 32 0
+      24.8366 0 16Z"
```

Comment:
Is this the circle around the exclamation mark? It is already implemented above (l.74)

Reply:
No, it's the little circle at the bottom of the exclamation mark.

Reply:
If you change the colors of the 2 paths to something other than red (each a different color) you can see that the first path is the full exclamation mark (little circle + vertical line) and the other one is an additional (opaque) circle ie the progress bar circle that is already implemented via `circle` above.

Reply:
Lol my bad, I just assumed. Will push an update...